Installation/rpm/arangodb.spec.in

```diff
@@ -186,9 +186,9 @@ mkdir -p %{buildroot}%{_piddir}
 %{_mandir}/*/arangobench*
 %{_mandir}/*/arangodump*
 %{_mandir}/*/arangoimport*
-%{_datadir}/arangodb3/js/common
-%{_datadir}/arangodb3/js/client
-%{_datadir}/arangodb3/js/node
+%{_datadir}/arangodb3/js/@CPACK_ARANGODB_JS_VERSION@/common
+%{_datadir}/arangodb3/js/@CPACK_ARANGODB_JS_VERSION@/client
+%{_datadir}/arangodb3/js/@CPACK_ARANGODB_JS_VERSION@/node
```

Comment:
Isn't the `js` part redundant now? I thought `js/` is already contained in `@CPACK_ARANGODB_JS_VERSION@`, or am I wrong?